src/store/metadata/base.py

```diff
@@ -86,3 +88,12 @@ def get_sub_topic(self, topic_id: str, sub_topic_id: str) -> Optional[Dict]:
         """
         Get a specific sub-topic for a specific topic
         """
+
+
+class ContextStore(ABC):
```

Comment:
this is clever but we don't need to inherit ABC, thats just for enforcing that the class that extends the class has the `@abstractmethod` decorated methods, not applicable here.

also can we move it to another file, say "context.py" or similar, just to avoid the implication from being in base.py that its an abstract class.